bin/write-good.js

```diff
@@ -43,134 +41,124 @@ var checks = [ // default checks
+const shouldParse = Object.keys(program).indexOf('parse') !== -1;
 
-var hasTextArg = args.some(function(arg) {
-  return arg.startsWith('--text')
-});
+const hasTextArg = args.some(arg => arg.startsWith('--text'));
 if (files.length === 0 && !hasTextArg) {
   console.log('You did not provide any files to check.');
   process.exit(1);
 }
 
-var hasChecks = args.some(function(arg) {
-  return arg.startsWith('--checks');
-});
+const hasChecks = args.some(arg => arg.startsWith('--checks'));
 
-var handleInvalidArgument = function(arg) {
-  console.log('"' + arg + '" is not a valid argument.');
+function handleInvalidArgument(arg) {
+  console.log(`"${arg}" is not a valid argument.`);
   process.exit(1);
 }
 
-if(hasChecks) {
+if (hasChecks) {
   // validate base args and infer valid check args from the imported custom check module
-  args.slice(1).forEach(function(arg) {
-    if(arg.startsWith('--text') || arg.startsWith('--checks') || arg === '--parse') return;
-    var isValid = Object.keys(opts.checks).some(function(check) {
-      return arg === '--' + check || arg === '--no-' + check;
-    });
-    if(!isValid) {
+  args.slice(1).forEach((arg) => {
+    if (arg.startsWith('--text') || arg.startsWith('--checks') || arg === '--parse') return;
+    const isValid = Object.keys(opts.checks).some(check => arg === `--${check}` || arg === `--no-${check}`);
+    if (!isValid) {
       handleInvalidArgument(arg);
     }
   });
 } else {
-  var optionArgs = args.filter(function(arg) {
-    return !files.includes(arg);
-  });
+  const optionArgs = args.filter(arg => !files.includes(arg));
   // validate default args if no custom checks module is provided
-  optionArgs.forEach(function(arg) {
-    if(arg.startsWith('--text')) return;
-    var isValid = program.options.some(function (option) {
-      return arg === option.long || arg === option.short;
-    });
-    if(!isValid) {
+  optionArgs.forEach((arg) => {
+    if (arg.startsWith('--text')) return;
+    const isValid = program.options.some(option => arg === option.long || arg === option.short);
+    if (!isValid) {
       handleInvalidArgument(arg);
     }
-  }); 
+  });
 }
 
-var include = true;
-var inputSupplied = files.length;
-var textInputSupplied = false;
+let include = true;
+let inputSupplied = files.length;
+let textInputSupplied = false;
 
-args.filter(function (arg) {
-  return arg.substr(0, 2) === '--';
-}).map(function (arg) {
-  return arg.substr(2);
-}).forEach(function (arg) {
+args.filter(arg => arg.substr(0, 2) === '--').map(arg => arg.substr(2)).forEach((arg) => {
   if (arg.indexOf('text=') !== -1) {
     opts[arg.substr(0, 4)] = arg.substr(5);
+    // eslint-disable-next-line no-multi-assign
     textInputSupplied = inputSupplied = true;
   } else if (arg.substr(0, 3) === 'no-') {
     opts[arg.substr(3)] = false;
   } else if (arg.substr(0, 4) === 'yes-') {
     opts[arg.substr(4)] = true;
-  } else if (arg == 'parse') {
+  } else if (arg === 'parse') {
+    // no opt
```

Comment:
Maybe add a TODO.

Reply:
I normally don't adds TODOs unless I am going to act on them. However, I'm fine doing this if you feel strongly about it

Reply:
I was thinking more that it would be good to have a TODO in the file, to which we could add an issue with a  'Good First Issue' label. Small things like this can help contributor engagement. It's not essential to this PR, though.